CHANGELOG.md

```diff
@@ -1,4 +1,21 @@
-## 2.17.1 (Unreleased)
+## 2.17.2 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-6.2.1913**
+- Supported Terraform version: **v0.12.x** and **v0.13.x**
+
+### Features:
+1. Implemented support for ``forward_traffic_to_transit`` in **aviatrix_site2cloud**
```

Comment:
1. Implemented further support for Custom Mapped and overlapping CIDR scenarios for **aviatrix_site2cloud** with attribute ``forward_traffic_to_transit``